include/wabt/ir.h

```diff
@@ -1245,6 +1245,12 @@ struct Module {
   BindingHash memory_bindings;
   BindingHash data_segment_bindings;
   BindingHash elem_segment_bindings;
+
+  // Feature tracking for a subset of supported features
```

Comment:
Perhaps be a little more explicit about why this separate from supported features.   How about "For a subset of features we track whether or not they are actually used in the module.  This is used by wasm2c to limit the output in certain cases."

And maybe call it `features_used` or something like that?

Reply:
Sure, done.